proto/pdpb.proto

```diff
@@ -799,3 +801,13 @@ enum QueryKind {
     Commit = 10;
     Rollback = 11;
 }
+
+message ReportBucketsRequest {
+    RequestHeader header = 1;
+
+    metapb.RegionEpoch region_epoch = 2;
+    metapb.Buckets buckets = 3;
+}
+
+message ReportBucketsResponse {
```

Comment:
how about carry RequestHeader?

Reply:
What is it used for?

Reply:
@bufferflies Do you mean `ResponseHeader`? It's a convention to carry potential error?

Reply:
yes, like cluster id is not right?